### DISCUSSION TITLE
Possibility of tabbed document views

### DISCUSSION THREAD

**Discussion by Jivix-zz** [KDE/ghostwriter#658] [Ideas]
Hi everyone,

I was wondering if anyone has worked on a large number of markdown files at the same time, such as when manually sorting the contents of one file into several others.

I think the most traditional solution to this would be to add tabbed browsing so manually handling a dozen or so windows is not required.

But I also had another idea. It would be interesting if you could Copy/Cut -> Append a snippet to another document wherever the cursor is currently placed there. This would be akin to Photoshop's cut and paste to another document. This may be slightly more use-case-specific than tabbed documents, but I think plenty of multi-doc users could find use for it. And it could coexist along side tabbed documents.

**Comment by gelatinbomb** [comment]
I really like this idea. I would like to have some tabs for handling some documents that I usually tend to use all at the same time.

**Comment by ghost** [comment]
Same here, I don't know the simplest way to implement it, but I kind of like the way VS Code has a "Open Folder" view, that way you can be working on multiple documents under a "root" folder. It seems you have a side buttons UI. Being able to display files on the side would be awesome!

**Reply by gelatinbomb** [reply]
Yeah that's a good option too!

**Comment by ghost** [comment]
@wereturtle Does this sound like a bad idea, or I'd love to know your thoughts

**Comment by Cultura-Difusa** [comment]
I think tabbed document/s would be great. Even better to be able to have something akin to Tmux (the terminal multiplexer).

I think multiplexing can enhance productivity at certain times (having real time split views for editing writing). Eg:

![tmux](https://user-images.githubusercontent.com/47992536/190666916-75b259d1-2275-4798-9bd6-2b0590b896ec.png)

**Comment by agaricus-campestris** [comment]
I really would love this feature, I feel this would be ok even for people not wanting to be distracted, maybe by making it possible to activate/deactivate tab view feature ? It would be interesting to have power users to comment on this so we here would know if this would be possible or not.

Thanks anyway to anyone reading this, and thanks a lot to anyone participating in ghostwriter, I may switch from Geany here =]

｡◕‿◕｡
Cheers,

**Comment by Valdus92** [comment]
I would love to see this as well. If not tabbed then opening several documents at once. Aren't we all scatter brains in this day and age of screen? Though I presume tabbed documents and multiple documents open are pretty much the same thing.

**Reply by Cultura-Difusa** [reply]
I think all bugs/requests have to be directed to kde-ghostwriter tracker/website. Not here, but here: https://invent.kde.org/office/ghostwriter/-/issues

Actually, now apparently you have to report here: https://bugs.kde.org/buglist.cgi?quicksearch=ghostwriter
